### PR TITLE
build molecule prefabs

### LINKED ISSUE #6: Build Molecule Prefabs
Need to build prefabs for each of the molecules which have been given to us.
Will require:
- Checking molecule sprites are properly centred to align with grid
- Prefabs for covalent interaction (pair of tiles) and pi bonds (individual tiles)
- Prefab for red blocking tiles 
- Object / collider for each benzene group which requires checking (might not need to be a collider but just an object so we can check the coordinates and compare to the tilemap?)

### PR DESCRIPTION
**Pull request by tcouch**
closes #6 

* Set pixels to units to 103 for all hex sprites
* Created prefabs for Pi, Cov and Steric interaction tiles with circle colliders
  * Tags are: PiInteraction, StericInteraction, CovalentInteractionL, CovalentInteractionR
  * Opted for two child objects for covalent representing left and right side. Should allow us to figure out which adjacent hex has to line up regardless of rotation?
* Create MoleculePart prefab to add to molecule hexes: empty object with a MoleculePart tag
* Created basic scene template with centre marker to help create molecule prefabs (same size as preview area)
* Set grid x-scale to 1 in level template
* Created prefabs for molecules 1,2, 3, 4 and 20 (the biggest)
* Written basic step by step instructions

### CODE REVIEW

File: Docs/creating_a_molecule_prefab.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Creating a molecule prefab
+
+* Open Molecule Layout Template Scene
+* Create Empty: Molecule_N (where N is a number)
+* In the inspector window, make sure the transform position is set to 0,0,0
+* Right click on Molecule_N and create empty called MoleculeSprite
+* Select MoleculeSprite and in the inspector window click Add Component
+* Search for and add a SpriteRenderer component
```
Comment (mmcleod89) [inline Docs/creating_a_molecule_prefab.md:8]: Can we prefab these steps, along with the empty objects for Pi/Covalent/Steric interactions?
We could have a molecule template which was a MoleculeSprite object with a sprite renderer and objects for each of the interactions already present as children, and then the user would just have to select the sprite and drag in the interactions that they want

Comment (mmcleod89) [reply Docs/creating_a_molecule_prefab.md:8]: (Not really necessary for this pull request and could be a separate issue)

**Review by mmcleod89 — APPROVED** [review]
Molecules look great, I think this structure will work. 
I'd definitely like us to have a molecule base  prefab so that 
- The team (and we!) don't have to recreate the same objects each time 
- We can attach scripts etc. to all the molecules at once without having to do it individually 
- This prefab can have a script attached at the top level that has a serialized field for the molecule sprite / pi interactions / covalent interactions / steric interactions sub parts which are then attached in the prefab, making them easier to access without having to search through Children and check tags and stuff. 

This can all be a separate PR though if you want to merge what you've done so far

File: Docs/creating_a_molecule_prefab.md (hunk 1 above)
Comment (tcouch) [reply Docs/creating_a_molecule_prefab.md]: Yeah, that's a much better way of doing it. Done, and updated mols 1-4 and 20.

**Pull request merged**